spec/mongoid/relations/embedded/one_spec.rb

```diff
@@ -1019,4 +1019,18 @@ class << person
       end
     end
   end
+
+  context "when parent validation of child is set to false" do
+
+    let(:building) do
+      building = Building.create
+      building.building_address = BuildingAddress.new
+      building.save
+      building.reload
+    end
+
+    it "parent successfully embeds an invalid child" do
+      expect(building.building_address).to be
```

Comment:
I'm trying to test that building.building_address is set, i.e. not nil. I could maybe do

expect(building.building_address).to_not be_nil             <- guessing on syntax, haven't tested

'.to be' seems to do what I want?